File: Problem-Solving/reverseString.py
# Method-1
def reverseStringOne(string):
    string = string[::-1]
    return string


# Method-2
def reverseStringTwo(string):
    reversedString = ""
    for i in string:
        reversedString = i + reversedString
    return reversedString


reverseStringOne("Hello")
reverseStringTwo("Hello")

File: Problem-Solving/reverseString.js
//Method-1

const reverseStringOne = (string) => {
  reversedString = string.split("").reverse().join("");
  console.log(reversedString);
};

//Mehtod-2
const reverseStringTwo = (string) => {
  let reversedString = "";
  for (let i = string.length - 1; i >= 0; i--) {
    reversedString += string[i];
  }
  console.log(reversedString);
};
reverseStringOne("Hello");
reverseStringTwo("Hello");
